requirements/common.txt

```diff
@@ -26,11 +27,15 @@ MarkupSafe==0.23
 mccabe==0.4.0
 pep8==1.7.0
 psycopg2==2.6.1
+pyaml==16.11.4
```

Comment:
PyYaml (http://pyyaml.org/wiki/PyYAMLDocumentation) is used (`import yaml`), but pyaml is a "PyYAML-based python module to produce pretty and readable YAML-serialized data" that is not used here (https://pypi.python.org/pypi/pyaml).